tests/test_clients.py

```diff
@@ -711,3 +711,35 @@ async def test_arxiv_doi_is_used_when_available() -> None:
     )
     assert result, "paper should be found"
     assert result.doi == "10.48550/arxiv.1706.03762"
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize(
+    ("doi", "score"),
+    [
+        ("10.1038/s41598-018-27044-6", 1),
+        ("10.1073/pnas.1205508109", 3),
+        ("10.1186/1471-2148-11-4", 2),
+        ("10.1016/j.semcdb.2016.08.024", 1),
+        ("10.1146/annurev.pathol.4.110807.092311", 2),
+        ("10.1016/j.bbcan.2023.188947", 1),
+    ],
+)
+@pytest.mark.asyncio
+async def test_tricky_journal_quality_results(doi: str, score: int) -> None:
```

Comment:
Can you either:
- Add a docstring just stating the meaning of "tricky"
- Add a comment (or `pytest.param` with `id="reason"`) next to each test case stating what's tricky